pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - Audio output
+  - CSS styling
+  - DOM tree
+---
+
+A _change in the content_ of a [web page][] caused by an event occurs when, by comparing the [web page][] before and after the event firing, at least one of the following occurs:
+
+- **visible changes:** the rendered pixels change in any part of the document that is currently within the [viewport][] or that can be brought into the [viewport][] via scrolling; or
+- **accessibility tree changes:** any state, property or event of a node representing an [accessible object][] of the [accessibility tree][] changes, or any node is inserted in, or removed from the [accessibility tree][]; or
+- **audible changes:** the audio rendered by the [web page][] changes.
+
+**Assumptions:**
+
+- If the [web page][] is rendering time-based media, rendered pixels and audio will be changing as part of the playback. The comparison in this instance should compare the pixels and audio that are rendered if the event is not fired, with the ones that are rendered if the event is fired.
+- For the purpose of this definition, it is assumed that the event causing the changes is the only origin of changes to the [web page][] and therefore the page does not change if there is no such event.
+- For the purpose of this definition, it is assumed that the changes happen instantly after the event firing and therefore the comparison between the page before and after the event firing can be made at any time after the firing of the event.
```

Comment:
Also, maybe sort of assuming that the event itself is instantaneous 🤔 
But that might be part of what we call "event" here.
So, an asynchronous request could be two events (making the request, getting an answer), or 1 non-instantaneous (making he request), or maybe 4 (starting to send the request, finishing sending, starting to receive the answer, finishing it).

---
Let's say that I can (maybe by tilting the phone) send a request for a bunch of data. After some times, I start receiving that data. It takes some time to download everything because I'm on an old EDGE network 📶 (or just because there is a lot of data) Once download is done, the phone takes some time processing it (e.g., computing some aggregates). Once this processing is done, the page is updated instantaneously (or at least faster than I can notice).

How many events are there in this scenario? Which one is causing the change in the page? Does the rule apply (or rather requires an UI control)?

Reply:
This is, for me, the biggest issue with the current definition... even moving to dom events we can have a device motion event that triggers an asynchronous call that will resolve only after 1 minute (or whatever other duration).
The only other option I see, is using the "10 minute" workaround. What do you think @Jym77?